templates/data-sources/project_ip_addresses.md.tmpl

```diff
@@ -0,0 +1,10 @@
+# {{.Type}}: {{.Name}}
```

Comment:
as autogenerated documentation is being add, would be good if we can add a step in as done [here](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/.github/workflows/code-health.yml#L76-L77). This ensures the resulting docs are aligned with the autogeneration.